File: solutions/beecrowd/1115/1115.py
import sys

for line in sys.stdin:
    x, y = map(int, line.split())

    if not x or not y:
        break

    if x > 0 and y > 0:
        print('primeiro')
    elif x < 0 and y < 0:
        print('terceiro')
    elif x < 0:
        print('segundo')
    else:
        print('quarto')

File: solutions/beecrowd/1115/1115.cpp
#include <cstdint>
#include <cstdio>

int main() {
    std::int16_t x, y;

    while (scanf("%d %d", &x, &y) != EOF && x != 0 && y != 0) {
        if (x > 0 && y > 0) {
            printf("primeiro\n");
        } else if (x < 0 && y < 0) {
            printf("terceiro\n");
        } else if (x < 0) {
            printf("segundo\n");
        } else {
            printf("quarto\n");
        }
    }

    return 0;
}
